code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -112,6 +112,12 @@ var/global/list/robot_modules = list(
 		T.degradation = 0 //We don't want robot tools breaking
 
 
+	for (var/obj/item/I in modules)
+		for (var/obj/item/weapon/cell/C in I)
+			C.charge = 999999999 //A quick hack to stop robot modules running out of power
```

Comment:
I don't know of a better way to do this but let's just make sure we don't forget this exists later.